src/homepageExperience/utils.ts

```diff
@@ -100,3 +100,26 @@ export const HOMEPAGE_NAVIGATION_STEPS_ARDUINO = [
     glyph: IconFont.StarSmile,
   },
 ]
+
+export const HOMEPAGE_NAVIGATION_STEPS_MQTT = [
+  {
+    name: 'Overview',
+    glyph: IconFont.BookOutline,
+  },
+  {
+    name: 'Connect to Broker',
+    glyph: IconFont.CopperCoin,
+  },
+  {
+    name: 'Subscribe to Topic',
+    glyph: IconFont.CopperCoin,
+  },
+  {
+    name: 'Define Data Parsing\n Rules',
+    glyph: IconFont.CopperCoin,
+  },
+  {
```

Comment:
I just want to say that there's still quite a bit of discussion going on about whether we want to go back to this 3 page design for MQTT. I would highly advise scaffolding this to use the _current_ MQTT single page design until that decision is made.

Reply:
thanks for providing that context @genehynson!

Reply:
+1

Reply:
Just got a memo from @Luckette that we plan on the multi-page design which is being drawn up by Tara in a figma right now.

Reply:
I heard otherwise from Barbara :/ Sorry not trying to be difficult - I just don't want to see you start down the wrong path here.

Reply:
Sounds good. Thank you for the context and I'll work on other things for now until we get an update. Thank!

Reply:
just fyi - I'll be OoO for a couple weeks. So feel free to resolve this comment and move on with whatever direction you get while I'm out